docs/manual.xml.head

```diff
@@ -16198,6 +16198,85 @@ bind index &gt; vfolder-window-forward
       </sect2>
     </sect1>
 
+    <sect1 id="pager-read-delay-feature">
+      <title>Pager Read Delay Feature</title>
+      <subtitle>Delay when the pager marks a previewed message as
+      read</subtitle>
+
+      <sect2 id="pager-read-delay-support">
+        <title>Support</title>
+        <para>
+          <emphasis role="bold">Since:</emphasis> NeoMutt 2021-06-??
```

Comment:
I didn't know what date to put here - it may depend on when you next cut a release

Reply:
put today's date, for now